vkconfig/mainwindow.cpp

```diff
@@ -78,9 +78,6 @@ MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent), ui(new Ui::MainWi
     ///////////////////////////////////////////////
     Configurator &configurator = Configurator::Get();
 
-    if (!configurator.VK_LAYER_PATH.isEmpty())
-        ui->groupBoxProfiles->setTitle("Vulkan Layers Configurations With VK_LAYER_PATH Precedence");
```

Comment:
In a previous review, Karl had said he thought this needed to be much more "in your face" when the VK_LAYER_PATH is in effect. I see you added it to the log, and that's good, I think it should go there too. Removing it from here might get some blow back. I don't strictly object to it's removal, but we did have some feedback that it needed to be more obvious to the user when in use, and this is one way to do that.